modules/ipi-install-node-requirements.adoc

```diff
@@ -8,39 +8,52 @@
 
 Installer-provisioned installation involves a number of hardware node requirements:
 
-- **CPU architecture:** All nodes must use `x86_64` CPU architecture.
+- *CPU architecture:* All nodes must use `x86_64` CPU architecture.
 
-- **Similar nodes:** Red Hat recommends nodes have an identical configuration per role. That is, Red Hat recommends nodes be the same brand and model with the same CPU, memory and storage configuration.
+- *Similar nodes:* Red Hat recommends nodes have an identical configuration per role. That is, Red Hat recommends nodes be the same brand and model with the same CPU, memory and storage configuration.
```

Comment:
...model with the same CPU, memory, and storage configuration.
(added a serial comma after _memory_)

Reply:
lgtm. added a few nits